### PR TITLE
Include form number at top of page (#2416)

### PR DESCRIPTION
**Pull request by cehsu**
Connects to department-of-veterans-affairs/vets.gov-team#2416
screen shot 2017-05-02 at 9 46 10 am

### CODE REVIEW

**Review by jbalboni — APPROVED** [review]
Looks good to me, feel free to merge it when you're ready. Make sure when you merge you choose the "Squash and merge" option in the PR.

File: src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx [hunk 1]
```diff
@@ -120,7 +122,7 @@ class EduBenefitsApp extends React.Component {
         {devPanel}
         <Element name="topScrollElement"/>
         <div className="usa-width-two-thirds medium-8 columns">
-          <FormTitle title="Apply for education benefits"/>
+          {!_.includes(['introduction'], endpoint) && <FormTitle title="Apply for education benefits" subTitle="Form 22-1990"/>}
```
Comment (jbalboni) [inline src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx]: It looks like this introduces a second way of telling if the page is the intro page (in addition to the later `endpoint === 'introduction'` check).

**Review by jbalboni — COMMENTED** [review]
Made some comments on your new changes.

File: src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx (hunk 1 above)
Comment (cehsu) [inline src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx]: Great idea @jbalboni, implemented in commit 49ca2dc.

File: src/js/edu-benefits/1990/containers/IntroductionPage.jsx [hunk 2]
```diff
@@ -10,6 +11,7 @@ class IntroductionPage extends React.Component {
       <div>
         <div className="row">
           <div className="small-12 columns">
+            <FormTitle title="Apply for education benefits"/>
```
Comment (cehsu) [inline src/js/edu-benefits/1990/containers/IntroductionPage.jsx]: Sounds good, implemented in commit 8991b53.

**Comment by jbalboni** [comment]
Not sure what happened here, but looks like there are a bunch of extraneous changes now.

**Comment by cehsu** [comment]
Thanks @jbalboni, extraneous changes tidied up.

**Pull request merged**